chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -146,6 +146,29 @@ if test "x$MD5_ADMIN" != "x$MD5_LOCAL"; then
     exit 1
 fi
 
+# Setup groups and users
+# # ---------------
+
+if [[ -z `grep '^qemu' /etc/group` ]]; then
+  groupadd --system --gid 484 qemu
+elif [[ -z `grep '^qemu:*.:484:*.' /etc/group` ]]; then
+  echo "Group qemu did not have gid 484."
+  exit 1
+fi
```

Comment:
Please use ```getent group qemu``` that's better than grepping through /etc/group.

And I think something like this would be a bit more readable:

```
gid=$(getent group qemu | cut -f 3 -d ":")
if [ -z "$gid" ]; then
   ...create group...
elif [ "$gid" != "484" ]; then
    ...error message...
    exit 1
fi
```